examples/flutter_gallery/lib/demo/material/slider_demo.dart

```diff
@@ -159,6 +159,50 @@ class _SliderDemoState extends State<SliderDemo> {
                 const Text('Continuous'),
               ],
             ),
+            Column(
+              mainAxisSize: MainAxisSize.min,
+              children: <Widget>[
+                Row(
+                  children: <Widget>[
+                    Expanded(
+                      child: Slider(
+                          value: _value,
+                          min: 0.0,
+                          max: 100.0,
+                          onChanged: (double value) {
+                            setState(() {
+                              _value = value;
+                            });
+                          }),
+                    ),
+                    Semantics(
+                      label: 'Editable numerical value',
+                      child: Container(
+                        width: 48,
+                        height: 48,
+                        child: TextField(
+                          onSubmitted: (String value) {
+                            final double myDouble = double.tryParse(value);
+                            if (myDouble != null &&
+                                myDouble >= 0 &&
+                                myDouble <= 100) {
+                              setState(() {
+                                _value = myDouble;
+                              });
+                            }
+                          },
+                          keyboardType: TextInputType.number,
+                          controller: TextEditingController(
+                            text: _value.toStringAsFixed(1),
```

Comment:
Why does this need a decimal place?

Reply:
My mistake. Material Design examples don't use decimals so... fixed.